docs/01-programs/02-candy-machine/10-how-to-guides/00-my-first-candy-machine.md

```diff
@@ -0,0 +1,600 @@
+
+In the current Alpha version you will have to modify the config file manually and add the guard configuration in the `guard` field. Here is an example how it can look like:
+
+```json
+{
+  "number": 10,
+  "symbol": "TEST",
+  "sellerFeeBasisPoints": 500,
+  "isMutable": true,
+  "isSequential": false,
+  "creators": [
+    {
+      "address": "DF2eiJojmgvcXDoQgSFzhrnt9Ds4GoYZmaCRMF8sWMYN",
+      "share": 100
+    }
+  ],
+  "uploadMethod": "bundlr",
+  "awsConfig": null,
+  "nftStorageAuthToken": null,
+  "shdwStorageAccount": null,
+  "pinataConfig": null,
+  "hiddenSettings": null,
+  "guards": {
+    "default": {
+      "mintLimit": {
+        "id": 1,
+        "limit": 3
+      },
+      "solPayment": {
+        "value": 0.5,
+        "destination": "Tes1zkZkXhgTaMFqVgbgvMsVkRJpq4Y6g54SbDBeKVV"
+      },
+      "startDate": {
+        "date": "2022-10-23T20:00:00Z"
+      }
+    }
+  }
+}
+```
+We added a default guard with the following settings:
+- `mintLimit` only allows minting a maximum of three NFTs (`limit`) per wallet.
+- `solPayment` requires a payment of 0.5 SOL (`value`) which is paid to the destination wallet (`destination`).
+- `startDate` restricts the mint function to not allow minting before `2022-10-23T20:00:00Z` (`date`)
+
+You can add multiple different guards e.g. to create multiple phases for OGs, WL and public mint. You can find more information about the possibilities on the [Guards](/programs/candy-machine/candy-guards) pages.
+
+After modifying and saving the manually modified config file we deploy the guard and set it as the guard of our candy machine by running:
+
+```bash
+sugar guard add
+```
+
+Afterwards it will only be possible to mint through that guard if all the restrictions are matched.
+
+<details>
+<summary>Output</summary>
+<p>
+[1/3] 🔍 Looking up candy machine
+
+Candy machine ID: Ews3L5NoAjjLEHYqEu47DqQ77nsqgNQs3NuELjBCd5bb
+
+[2/3] 🖥  Loading candy guard
+▪▪▪▪▪ Done
+
+Candy guard ID: 7XJdGaywrtcEpohrcZ7kYeMjEHNFVq1XjVpBQ6Doi7TP
+
+[3/3] 📦 Wrapping
+Signature: 3C7iXdu9msYviFAJz4esMvkMAiSXTfExBYr9E5VEJfrmJnojrfoEEDPGzJ7qGDx1J5Pf6bHZtfRqYvBPLrc35on5
+
+The candy guard is now the mint authority of the candy machine.
+
+✅ Command successful.
+</p>
+</details>
+
+If required you can modify the guard afterwards by running:
+
+```bash
+sugar guard update
+```
+
+To see the current settings like if you want to change the start date you can run:
+
+```bash
+sugar guard update
+```
+
+<details>
+<summary>Output</summary>
+<p>
+[1/2] 🔍 Loading candy guard
+▪▪▪▪▪ Done
+Candy guard ID: 7XJdGaywrtcEpohrcZ7kYeMjEHNFVq1XjVpBQ6Doi7TP
+
+[2/2] 🖥  Updating configuration
+Signature: 5nQ1dasS3QAQJH7fBwFXd2VgyRirfYmnZ8XKgsfvWubX4hJ6jzY4dzqEVpXpULgw1PsC5eyJriSMCnVXxw2Mckae
+
+✅ Command successful.
+</p>
+</details>
+
+
+## Allow users to mint
```

Comment:
I assumed this section would be part of this guide tbh.

I think this is a very important part of setting up a Candy Machine but I understand that the size of this guide is already pretty big.

How do you feel about writing a "Part 2" that focuses on the minting side using the JS SDK? We could rename this guide: "**How to Create a Candy Machine v3 - Part 1 (Sugar)**" and create a new guide called "**How to Create a Candy Machine v3 - Part 2 (JS SDK)**".

I'm happy to merge Part 1 first and do Part 2 in another PR. That being said we should already have the page created with a "coming soon" placeholder as part of this PR.

Reply:
Makes sense! No sure how to best structure the minting how to, but it should be included.